src/txmempool.h

```diff
@@ -78,7 +80,8 @@ class CTxMemPoolEntry
 
 public:
     CTxMemPoolEntry(const CTransaction& _tx, const CAmount& _nFee,
-                    int64_t _nTime, double _dPriority, unsigned int _nHeight, bool poolHasNoInputsOf = false);
+                    int64_t _nTime, double _dPriority, unsigned int _nHeight, bool poolHasNoInputsOf = false,
+                    unsigned int nSigOps = 0);
```

Comment:
OK agreed.  The default is just for the tests, I think for now it makes sense to remove all defaults from the constructor and just make the tests fill them in.  Unfortunately there are 2 other pulls open which also add arguments to the constructor and require changing the tests.  I'm going to hold off updating the tests in this pull until we're a bit closer assuming those might be merged first.